sdk/keyvault/Azure.Security.KeyVault.Administration/src/Azure.Security.KeyVault.Administration.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+    <Description>This is the Microsoft Azure Key Vault Administration client library</Description>
+    <AssemblyTitle>Microsoft Azure.Security.KeyVault.Administration client library</AssemblyTitle>
+    <Version>4.1.0-preview.1</Version>
```

Comment:
Do we want to start the version here? Is that to align with the other keyvault libraries?

Reply:
Yes, to align with other Key Vault packages.